aten/src/ATen/AccumulateType.cpp

```diff
@@ -15,7 +15,7 @@ c10::ScalarType toAccumulateType(c10::ScalarType type, c10::DeviceType device) {
           return CppTypeToScalarType<at::acc_type_device<scalar_t, c10::DeviceType::CPU>>::value;  \
       }
 
-    AT_FORALL_SCALAR_TYPES_WITH_COMPLEX_EXCEPT_COMPLEX_HALF(DEFINE_CASE)
+    AT_FORALL_SCALAR_TYPES_WITH_COMPLEX_EXCEPT_COMPLEX_HALF_F8NZ(DEFINE_CASE)
```

Comment:
Can you please explain why this one is needed?

Reply:
I changed the name of the `AT_FORALL_SCALAR_TYPES_WITH_COMPLEX_EXCEPT_COMPLEX_HALF` macro to make it clear it isn't all scalar types. Unfortunately, were I to add F8NZ types to that macro, I'd have to add CUDA_ACC_TYPEs and MPS_ACC_TYPEs for these.